docs/resources/Auto_Moderation.md

```diff
@@ -73,13 +73,13 @@ Characterizes the type of content which can trigger the rule.
-| keyword_filter      | array of strings *                                                                                                | KEYWORD                  | substrings which will be searched for in content (Maximum of 1000)                        |
-| regex_patterns      | array of strings **                                                                                               | KEYWORD                  | regular expression patterns which will be matched against content (Maximum of 10)         |
-| presets             | array of [keyword preset types](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-keyword-preset-types) | KEYWORD_PRESET           | the internally pre-defined wordsets which will be searched for in content                 |
-| allow_list          | array of strings *                                                                                                | KEYWORD_PRESET           | substrings which will be exempt from triggering the preset trigger type (Maximum of 1000) |
-| mention_total_limit | integer                                                                                                           | MENTION_SPAM             | total number of unique role and user mentions allowed per message (Maximum of 50)         |
+| Field               | Type                                                                                                              | Associated Trigger Types | Description                                                                       |
+| ------------------- | ----------------------------------------------------------------------------------------------------------------- | ------------------------ | --------------------------------------------------------------------------------- |
+| keyword_filter      | array of strings *                                                                                                | KEYWORD                  | substrings which will be searched for in content (Maximum of 1000)                |
+| regex_patterns      | array of strings **                                                                                               | KEYWORD                  | regular expression patterns which will be matched against content (Maximum of 10) |
+| presets             | array of [keyword preset types](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-keyword-preset-types) | KEYWORD_PRESET           | the internally pre-defined wordsets which will be searched for in content         |
+| allow_list          | array of strings *                                                                                                | KEYWORD, KEYWORD_PRESET  | substrings which will be exempt from triggering the rule (Maximum of 1000)        |
```

Comment:
the maximum allowlist word count for `KEYWORD` will be 100. Maybe we can just say Maximum of 100 or 1000 here? and clarify in the existing footnote below?

Reply:
oh its a shared footnote with `keyword_filter` field. hmm maybe create a new dedicated footnote for the `allow_list` field. Something like this:

```
An allow_list string can be a phrase which contains multiple words. Wildcard symbols can be used to customize how each string will be matched. Rules with `KEYWORD` [trigger_type](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-trigger-types) accept a maximum of 100 strings. Rules with `KEYWORD_PRESET` [trigger_type](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-trigger-types) accept a maximum of 1000 strings.
```

The other change is that allow list keywords *can* contain wildcard symbols now, so we can remove the `(not available to allow lists)` qualification in the current footnote.

Reply:
don't want to create too much work for you though, I was about to put up a change that included this + something else and searched for existing PR's and found yours. How about I just finish making all the additional changes on my PR and we close this one out to make it easier?